src/App.tsx

```diff
@@ -5,7 +5,16 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = (newMovie: Movie) => {
```

Comment:
If you put down your function - will be better use useCallback
```suggestion
  const addMovie = useCallback((newMovie: Movie) => {
```